addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -520,7 +520,7 @@ public void shutdown() {
         try {
             jettyClient.stop();
         } catch (Exception e) {
-            logger.error(e.getMessage());
+            logger.error("{}", e.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace